data/104/2210/en_US.json

```diff
@@ -1,11 +1,11 @@
 {
 	"104": {
 		"2210": {
-			"name": "Unknown",
-			"message": "",
+			"name": "STOPPED_BY_POLICYLIST",
```

Comment:
Where did this name come from?

Reply:
Quarky when they were figuring out how to fix 104-2210. But I have been attempting to figure out how to convert results to error codes on undocumented modules, which I haven't made much progress on in recent weeks as I'm completely stuck